---
string current_place
int64 robot_id
